cmd/image-builder/repos.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"github.com/osbuild/images/pkg/reporegistry"
+)
+
+// XXX: copied from "composer", should be exported there so
+// that we keep this in sync
+// XXX2: means we need to depend on osbuild-composer-common or a new rpm
+// that provides the relevant packages *or* we use go:embed (cf images#1038)
+var repositoryConfigs = []string{
```

Comment:
Food for thought: Should we stop using our custom repo format, and start using the same format as `dnf` uses in `/etc/yum.repos.d`? Our depsolver should be already able to consume via the `root` opion. Definitely not something to address in this PR, but I think it's a good conceptual dicusssion.

Reply:
I have no strong opinion here, I would like to centralize what we have though. I would like to be able to build all supported imagetypes from this binary, ideally without much/any external dependencies. I was thinking go:embed for this and having the list of supported reos in images and the ability to override. But there is some pushback for this idea (not sure why though, I'm probably missing some details why this would be an issue).